VulkanHppGenerator.cpp

```diff
@@ -865,6 +877,7 @@ struct VendorIDData
 
 struct VkData
 {
+  std::map<std::string, AliasData>              aliases; // Enum Aliases
```

Comment:
Why not to use `std::unordered_map<std::string, xxx>` since it has better performance than `std::map<std::string, xxx>`?

Reply:
Both, std::map and std::unordered_map, may have benefits here. std::map is faster with regards to insertion complexity (depends on allocation performance), std::unordered_map is faster with regards to lookup complexity (depends on speed up hash-computation for the string). So choosing the one or the other won't change much.